perf/scripts/get_cpuinfo.py

```diff
@@ -1,4 +1,25 @@
 import subprocess
+import platform
+from shutil import which
+def getestimatedcpufrequency():
+    # We should be checking the architecture we are running on, but platform.processor()
+    # is "unknown" in our container
+    if platform.system() != "Linux":
+        return None
+    p = subprocess.run(["gcc", "-O2", "-o", "/opt/test/cpu_frequency_estimate", "/opt/test/cpu_frequency_estimate.c"])
```

Comment:
This cannot work in the profiling docker image until gcc is installed. Is that intentional? If so, please adapt Dockerfile (also for x64) to enable testing.

Reply:
This should be resolved! Sorry, I forgot to update that Dockerfile. :)